.circleci/config.yml

```diff
@@ -139,6 +133,9 @@ jobs:
             sleep 10  # Give the services enough time to start (or fail)
 
             for c in as1301 as1303 as1305; do
+              # XXX(matzf) DEBUG
+              docker-compose exec $c /bin/bash -c \
+                'cat /var/log/scion/*'
```

Comment:
```suggestion
                'cat /var/log/scion/*'
```

Reply:
Sorry, is there a diff? :see_no_evil:

Reply:
Sorry, misquoted: Do you want to leave this in, since:
># XXX(matzf) DEBUG

i.e. cat all the logs? Might be too verbose to be useful

Reply:
Ah. Yes, removed